### PR TITLE
Update `osu!mania Malaysia Tournament 4` to Finals

### PR DESCRIPTION
**Pull request by fajar13k**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/OMMT/4/en.md [hunk 1]
```diff
@@ -107,6 +107,35 @@ The osu!mania Malaysia Tournament 4 is run by various community members.
 
 *Note: Maps with asterisk in its ordering list is considered to be MAD map. It is a twist from the organiser to make certain map unbannable, and in some occasion will force players to use modifier to play along with it. Each stage may have different kind of MAD twist, details are written for each stage.*
```
Comment (Walavouchey) [inline wiki/Tournaments/OMMT/4/en.md:108]: there are no asterisks (`*`) for any of the MAD maps

File: wiki/Tournaments/OMMT/4/en.md [hunk 2]
```diff
@@ -107,6 +107,35 @@ The osu!mania Malaysia Tournament 4 is run by various community members.
...
 
+### Finals
+
+Finals' MAD map is **SV2**, the twist is it has to be played under storyboard turned on, and 0% background dim.
```
Comment (Walavouchey) [inline wiki/Tournaments/OMMT/4/en.md:112]:
```suggestion
The Finals' MAD map is **SV2**. The twist is it has to be played with storyboard turned on and 0% background dim.
```

File: wiki/Tournaments/OMMT/4/en.md (hunk 1 above)
Comment (fajar13k) [reply wiki/Tournaments/OMMT/4/en.md:108]: At the very end, should I add it to the first line instead?
I missed the SV2 in Finals though.

File: wiki/Tournaments/OMMT/4/en.md [hunk 3]
```diff
@@ -107,6 +107,35 @@ The osu!mania Malaysia Tournament 4 is run by various community members.
...
+  3. [Sot-C - Extra Stage (TheFunk) \[Bonus Round\]](https://osu.ppy.sh/beatmapsets/2105933#mania/4418876)
+- SV
+  1. [shadow\_bling - Crush Alcohol (awdse22) \[DRUNK DRIVING IN POLICE CAR\]](https://osu.ppy.sh/beatmapsets/2226663#mania/4723525)
+  2. [ZxNX - FORTALiCE (Promachos) \[DiSARRAY\]](https://osu.ppy.sh/beatmapsets/2092706#mania/4387297)
+- Tiebreaker
+  1. **[Sydosys - ArchangeL (ERA Imperial) \[Final Judgment\]](https://osu.ppy.sh/beatmapsets/2185088#mania/461886)**
+
 ### Semifinals
 
 Semifinals' MAD map is **RC5**, the twist is it has to be played with **Random** modifier.
```
Comment (Walavouchey) [inline wiki/Tournaments/OMMT/4/en.md:141]:
```suggestion
The Semifinals' MAD map is **RC5**. The twist is it has to be played with the *Random** mod.
```

File: wiki/Tournaments/OMMT/4/en.md (hunk 1 above)
Comment (fajar13k) [reply wiki/Tournaments/OMMT/4/en.md:108]: Applied

File: wiki/Tournaments/OMMT/4/en.md [hunk 4]
```diff
@@ -163,10 +192,10 @@ Quarterfinals' MAD map is **RC4**, the twist is it has to be played with **Doubl
 
 ### Round of 16
 
-Round of 16's MAD map is **LN3**, the twist is the map has reverse shield patterns with little to no spacing, to balance it, OD0 was set for the map.
-
 **[Download the mappack here (70 MB)](https://cdn.discordapp.com/attachments/1257367722136703087/1262440243164876902/bomb16.zip?ex=66974358&is=6695f1d8&hm=a3a1f2b1c9a6c95087a670093e210a5bab23026440ad6196f5df35487281bed1&)**
```
Comment (Walavouchey) [inline wiki/Tournaments/OMMT/4/en.md:195]: please don't use discord cdn links <.<
they expire after a while

File: wiki/Tournaments/OMMT/4/en.md [hunk 5]
```diff
@@ -188,10 +217,10 @@ Round of 16's MAD map is **LN3**, the twist is the map has reverse shield patter
 
 ### Round of 32
 
-Round of 32's MAD map is **RC3**, the twist is it has to be played with **HardRock** modifier.
-
 **[Download the mappack here (94 MB)](https://cdn.discordapp.com/attachments/1257367722136703087/1259895755179556937/bomb.zip?ex=668e019b&is=668cb01b&hm=0e312cd523c626b8f5c647df75c953dc5eef7f7946e5b8b6d6335b10c3791211&)**
```
Comment (Walavouchey) [inline wiki/Tournaments/OMMT/4/en.md:220]: same here

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
please apply @Walavouchey 's latest review in a future update, not going to block this PR though since you didn't introduce any new issues w.r.t that

**Pull request merged**